Assets/Scripts/Communication/Communication.cs

```diff
@@ -93,8 +89,8 @@ public static bool Connect(Uri url)
 		return m_IsConnected;
 	}
 
-	public static bool IsConnected(){
-		return m_Socket != null && m_Socket.IsAlive && m_IsConnected;
+	public static bool IsConnected {
+		get{return m_Socket != null && m_Socket.IsAlive && m_IsConnected;}
```

Comment:
Can we not make this a getter since it is doing more than just getting a variable value?

Reply:
yupppppp